framework/lib/components/combo-picker/combo-picker-field.tsx

```diff
@@ -44,14 +81,11 @@ export const ComboPickerField = ({
     isRequired={ isRequired }
     validate={ validate }
   >
-    { ({ value, onChange }) => (
+    { ({ value }) => (
       <ComboPicker
         data-testid="combo-picker-test-id"
         aria-label={ label }
-        onChange={ (comboPickerValue) => {
-          onChange(comboPickerValue)
-          onChangeCallback?.(comboPickerValue)
-        } }
+        onChange={ onChange }
```

Comment:
@richardilleskosmala Could you please explain the reasoning behind this change? I'm concerned it might introduce a bug

Reply:
Changed back! thanks for bringing it up :)